### ISSUE TITLE
MATSim Accessibility extension plugin

### ISSUE THREAD

**Issue by stephenk80** [matsim-org/matsim-code-examples#435]
Hello,
I need help on how to use the MATSim accessibility extension as described in the MATSim book.  I need to apply it to my masters' project. Basics on how to set-up the accessibility extension and to run the grid-based and zone-based examples as described in the MATSim book will be appreciated. 

@dziemke, @kainagel  

Thank you in advance,
StephenK

**Comment by prvnpandey** [comment]
Every extension (plugin), written comes with java class from where you can run. Could you specify in particular which extension you are trying to use? In order to use this advanced matsim, you should know understanding about java.

**Comment by stephenk80** [comment]
Hi prvnpandey,

Thank you for your response. 
In the MATSim book, it states that the extensions can be accessed on the website but if i check on the website there is no extension for Accessibility. If I check on the libraries I cannot find it as well. I am lost. Your assistance is much appreciated. Please see attached images. 
![Matsim 1](https://user-images.githubusercontent.com/71983893/94670174-832d2280-0312-11eb-8bd9-06c8dca377dd.jpg)
![Matsim 2](https://user-images.githubusercontent.com/71983893/94670204-8c1df400-0312-11eb-9b6f-67478ab50128.jpg)
![Matsim 3](https://user-images.githubusercontent.com/71983893/94670231-92ac6b80-0312-11eb-9e36-f3423717623c.jpg)

**Comment by prvnpandey** [comment]
How did you clone the libraries from GitHub, there should to a folder of maven dependencies in which you can find all the accessibility libs.
![image](https://user-images.githubusercontent.com/38705566/94814616-e04fd380-03f9-11eb-9f34-eaad27a5faf1.png)

**Comment by JWJoubert** [comment]
You should the `accessibility` contrib as a Maven dependency. Depending on the version you are running (I am assuming the latest release 12.0), it will be something like

```
repositories>
  <repository>
    <id>matsim</id>
    <name>MATSim release repository</name>
    <url>http://dl.bintray.com/matsim/matsim</url>
  </repository>
</repositories>

<dependencies>
  <dependency>
    <groupId>org.matsim.contrib</groupId>
    <artifactId>accessibility</artifactId>
    <version>12.0</version>
  </dependency>
</dependencies>
```

**Comment by stephenk80** [comment]
Much appreciated @prvnpandey and @JWJoubert.

**Comment by stephenk80** [comment]
I am trying to run the RunAccessibilityExample. The config file that I downloaded from the libraries is not working. Is there any updated config file and facilities files for the RunAccessibilityExample? See attached: the location where I got the files, the structure of the config and facilities files as well as the warning errors. 

Thank you.


![File location](https://user-images.githubusercontent.com/71983893/95672216-39103080-0b9f-11eb-8f0e-ab78143230d3.jpg)

![config image](https://user-images.githubusercontent.com/71983893/95672310-087cc680-0ba0-11eb-98f1-10c681caef4e.jpg)

![facilities image](https://user-images.githubusercontent.com/71983893/95672323-19c5d300-0ba0-11eb-902d-2ef838a8d688.jpg)

![Warning Errors](https://user-images.githubusercontent.com/71983893/95672325-1d595a00-0ba0-11eb-830f-8d55ed3df117.jpg)

**Comment by dziemke** [comment]
Try to comment (i.e. deactivate) the line "accessibilityDestinationSamplingRate" in the config file and re-run.

**Comment by stephenk80** [comment]
I have deactivated the line "accessibilityDestinationSamplingRate", after running the new config file, I am getting a new error. In this case, the error is coming from "cellSizeForCellBasedAccessibility". From the config file it is assigned a value of 100. Please see the attached images. 


![New Error](https://user-images.githubusercontent.com/71983893/95758047-39482300-0ca8-11eb-8fcd-ee6964fcd269.jpg)

![config value](https://user-images.githubusercontent.com/71983893/95758033-30575180-0ca8-11eb-821a-585af54ad2c3.jpg)

**Comment by dziemke** [comment]
Seems like that example config file hadn't been touched in a while.
Two other minor alterations were necessary.
I just included them and will merge them into the master once the tests have passed.

**Comment by stephenk80** [comment]
Noted - thank you @dziemke

**Comment by stephenk80** [comment]
@dziemke, I am making a follow-up on the config file. I need to know if it is now updated so that I try a few examples.
